src/gui/order_widget.py

```
@@ -1,11 +1,23 @@
+    QWidget,
+    QVBoxLayout,
+    QHBoxLayout,
+    QTableWidget,
+    QTableWidgetItem,
+    QPushButton,
+    QLabel,
+    QMessageBox,
+)
 from PySide6.QtCore import Qt, Slot
 
+from ..printer.manager import PrinterManager
```

Pylint (3.3.7)

[error] Attempted relative import beyond top-level package

(E0402)